Makefile

```diff
@@ -1,5 +1,5 @@
 CI_BUILD_NUMBER ?= $(USER)-snapshot
-VERSION ?= 2.4.$(CI_BUILD_NUMBER)
+VERSION ?= 2.5.$(CI_BUILD_NUMBER)
```

Comment:
correct - async routing is an opt-in feature and existing routing config _should_ continue to work normally